src/account/account.cairo

```diff
@@ -27,6 +25,10 @@ mod Account {
     use starknet::get_contract_address;
     use starknet::get_tx_info;
 
+    const TRANSACTION_VERSION: felt252 = 1;
+    // 2**128 + TRANSACTION_VERSION
+    const QUERY_VERSION: felt252 = 0x100000000000000000000000000000001;
```

Comment:
huh is this the same as `340282366920938463463374607431768211457`? way better representation if so

Reply:
It is, yes